actions/ci-beholder-validator/action.yml

```diff
@@ -0,0 +1,101 @@
+name: "Schema Validator Action"
+description: "Validates schema compatibility with schema registry"
+
+inputs:
+  checkout-repo:
+    description: "Whether to checkout the repository"
+    required: false
+    default: "true"
+  checkout-repo-fetch-depth:
+    description: "Fetch depth for checkout"
+    required: false
+    default: "0"
```

Comment:
You can remove this and the first step of the action if you want. It doesn't really add much value, especially if you're checking out the repo before calling this action anyways.

Reply:
will do